tests/test_plot.py

```diff
@@ -0,0 +1,391 @@
+    img_name = input["save"]["as"]
+
+    dgpost.utils.plot(**input)
+    are_images_equal(img_name, f"ref.{img_name}")
+
+
+# "test.df.pkl",
+# "ndot.units.ufloat.df.pkl",
+# "xinxout.ufloat.df.pkl",
+@pytest.mark.parametrize(
+    "data_file, input",
+    [
+        (
+                "flowcx.units.ufloat.df.pkl",
+                {
+                    "ax_args": [
+                        {
+                            "series": [
+                                {
+                                    "y": "cin->*",
+                                    "colors": ["b", "r", "g"],
+                                    "kind": "scatter",
+                                },
+                            ],
+                            "methods": {
+                                "tick_params": {
+                                    "labelbottom": False
+                                },
+                            },
+                            "rows": (0, 1),
+                            "ylabel": "$C_{in}$",
+                            "legend": True,
+                        },
+                        {
+                            "series": [
+                                {
+                                    "y": "xout->*",
+                                    "colors": ["b", "r", "g", "y", "brown"],
+                                    "kind": "scatter",
+                                },
+                            ],
+                            "methods": {
+                                "tick_params": {
+                                    "labelbottom": False
+                                },
+                            },
+                            "rows": (1, 2),
+                            "ylabel": "$X_{out}$",
+                            "legend": True,
+                        },
+                        {
+                            "series": [
+                                {
+                                    "y": "flowin",
+                                    "color": "blue",
+                                    "kind": "line",
+                                },
+                                {
+                                    "y": "flowout",
+                                    "color": "red",
+                                    "kind": "line",
+                                },
+                            ],
+                            "rows": (2, 3),
+                            "xlabel": "index",
+                            "ylabel": "$flow$",
+                            "legend": True,
+                        },
+                    ],
+                    "nrows": 3,
+                    "style": test_style_1,
+                    "fig_args":
+                        {
+                            "figsize": (12, 12),
+                        },
+                    "save": {"as": "test.flowcx.png", "tight_layout": {}},
+                },
+        ),
+    ]
+)
```

Comment:
Added new example with the new feature to select multiple columns by "*" and also to test the selection of multiple properties with it (i.e. `colors`).
Also small test for the labels and legend.
(It is not so polished atm but wanted to push it before I leave)